README.md

```diff
@@ -62,7 +62,7 @@ The balance not covered by hard deposits can be withdrawn by the issuer at any t
 
 #### Signatures
 
-For signing purposes the fields are encoded in **packed encoding** in the order specified below. `SimpleSwap` uses the same signing scheme as `eth_sign` (with the `Ethereum Signed Message` prefix).
+For signing purposes the chequebook uses EIP-712 Ethereum typed structured data hashing and signing. The `EIP712Domain` domain name is `ERC20SimpleSwap` in version `1.0`. The `chainId` field is used, the `verifyingContract` is omitted.
```

Comment:
EIP says: `address verifyingContract the address of the contract that will verify the signature. The user-agent may do contract specific phishing prevention.`

Could you help me to explain this? Why don't we need it?

Reply:
added the `salt` reference. we don't need the `verifyingContract` because the address is already contained in the cheque (and all other signed structures).

Reply:
changed the domain name to `Chequebook`